packages/ruby/spec/readme/payload_spec.rb

```diff
@@ -1,53 +1,53 @@
-require "readme/payload"
-require "uuid"
+require 'readme/payload'
+require 'uuid'
+
+har_json = File.read(File.expand_path('../../fixtures/har.json', __FILE__))
 
 RSpec.describe Readme::Payload do
-  before :each do
-    har_json = File.read(File.expand_path("../../fixtures/har.json", __FILE__))
-    @har = double("har", to_json: har_json)
-    @uuid = UUID.new
-  end
+  let(:har) { double('har', to_json: har_json) }
+
+  uuid = UUID.new
```

Comment:
I would put this in a `let` for the sake of good practices. Outside of the `let` it could maybe introduce leakage, maybe. 

`let`'s are nice because they're lazy, so there's no overhead if it's not used everytime.

Reply:
Tried this yesterday with `let(:uuid) { UUID.new }` but it's throwing this exception when trying to do `uuid.generate` in a test:

![screen_shot_2022-08-11_at_3 10 10_pm_720](https://user-images.githubusercontent.com/33762/184251470-56145bc9-0a43-42b7-98ac-265fb99d7aff.png)

Since that's the only test that's using this UUID instance I'm going to move `UUID.new` into that so it's calling `UUID.new.generate`.